pycqed/analysis/fitting_models.py

```diff
@@ -237,6 +237,62 @@ def SlopedHangerFuncAmplitude(f, f0, Q, Qe, A, theta, slope):
     return np.abs((1.+slope*(f/1.e9-f0)/f0) *
                   HangerFuncAmplitude(f, f0, Q, Qe, A, theta))
 
+def SlopedHangerFuncAmplitudeGuess(data, f, fit_window=None):
+    xvals = f
+    # print(f)
+    # print(np.squeeze(data))
+    peaks = a_tools.peak_finder(xvals, data)
+      # Search for peak
+    if peaks['dip'] is not None:    # look for dips first
+        f0 = peaks['dip']
+        amplitude_factor = -1.
+    elif peaks['peak'] is not None:  # then look for peaks
+        f0 = peaks['peak']
+        amplitude_factor = 1.
+    else:                                 # Otherwise take center of range
+        f0 = np.median(xvals)
+        amplitude_factor = -1.
+        # logging.error('No peaks or dips in range')
+        # If this error is raised, it should continue the analysis but
+        # not use it to update the qubit object
+    min_index = np.argmin(data)
+    max_index = np.argmax(data)
+    min_frequency = xvals[min_index]
+    max_frequency = xvals[max_index]
+    # Fit data according to the model required
+    # added reject outliers to be robust agains CBox data acq bug.
+    # this should have no effect on regular data acquisition and is
+    # only used in the guess.
+    amplitude_guess = max(dm_tools.reject_outliers(data))
+
+    # Creating parameters and estimations
+    S21min = (min(dm_tools.reject_outliers(data)) /
+              max(dm_tools.reject_outliers(data)))
+
+    Q = f0 / abs(min_frequency - max_frequency)
+
+    Qe = abs(Q / abs(1 - S21min))
+    guess_dict = {'f0': {'value': f0*1e-9,
```

Comment:
I think there should not be factors of 1e-9 (or 1e9) at this level. 
As this is mostly a historical thing I suggest we leave it for now but to prevent nasty bugs resulting from conversion we should always use SI units in the code. Special plotting and printing functions exist to ensure that when we display it we get the right prefixes. 

For now I suggest we leave it but you should know about this.